spec/bundle/locker_spec.rb

```diff
@@ -40,6 +40,17 @@
     end
   end
 
+  describe ".whalebrew_list" do
+    before do
+      allow(Bundle::WhalebrewDumper).to receive(:images).and_return(["whalebrew/wget"])
+      allow(locker).to receive(:`).with("docker image inspect whalebrew/wget --format '{{ index .RepoDigests 0 }}'").and_return("whalebrew/wget@sha256:abcd1234")
```

Comment:
```suggestion
      allow(locker).to receive(:`).with("docker image inspect whalebrew/wget --format '{{ index .RepoDigests 0 }}'")
                                  .and_return("whalebrew/wget@sha256:abcd1234")
```

or whatever doesn't wrap in the GitHub diff UI